README.md

```diff
@@ -1,32 +1,22 @@
 Caddy Module: http.handlers.cache
 ================================
 
-**⚠️ Work-in-progress**
-
-This is a distributed HTTP cache module for Caddy.
+This is a distributed HTTP cache module for Caddy based on [Souin](https://github.com/darkweak/souin) cache.
 
 ## Features
 
-* Supports most HTTP cache headers defined in [RFC 7234](https://httpwg.org/specs/rfc7234.html) (see the TODO section for limitations)
-* Sets [the `Cache-Status` HTTP Response Header](https://httpwg.org/http-extensions/draft-ietf-httpbis-cache-header.html)
+ * Supports most HTTP cache headers defined in [RFC 7234](https://httpwg.org/specs/rfc7234.html) (see the TODO section for limitations)
+ * Sets [the `Cache-Status` HTTP Response Header](https://httpwg.org/http-extensions/draft-ietf-httpbis-cache-header.html)
+ * REST API to purge the cache and list stored resources.
+ * Builtin support for distributed cache.
+
+
+## Example Configurations
```

Comment:
Once decisions on feature set are decided, a snippet Caddyfile example in the README is usually better to present to users to get a feel for it.

Links can demonstrate more verbose setups/features.